### ISSUE TITLE
Daemon issue

### ISSUE THREAD

**Issue by sc0ch** [citicashio/citicash#3]
![image](https://user-images.githubusercontent.com/5249710/47317975-7ea25e80-d653-11e8-9c30-0d5def7384e2.png)

RPC wallet crashing after...

**Comment by salda** [comment]
Thanks for reporting, I have seen it only once in the past. 
I am trying to solve it now. Do you know the steps, how to reproduce it?

**Comment by salda** [comment]
Is the daemon the same version as the RPC-wallet? 
I have for example "Citicash 'Itynka' (v0.0.3.2-a81828a)" both (it is displayed in the daemon, when you call "help").

**Comment by sc0ch** [comment]
> Thanks for reporting, I have seen it only once in the past.
> I am trying to solve it now. Do you know the steps, how to reproduce it?

Ubuntu 18.04.1, citicash compiled from source (NOT git clone), boost 1.65.1.

Same version of course.

**Comment by mzk** [comment]
fixed in ce46d047d08032fa307cdd434ee7a27dbc1dd90b

**Comment by mzk** [comment]
https://github.com/citicashio/gui-wallet-win/commit/37813719bbdb681e819a1d4f2d7ba36903265ad0

**Comment by salda** [comment]
Thanks. Working on it now. Duplicate of #4
